salt/modules/status.py

```diff
@@ -1054,8 +1054,12 @@ def master(master=None, connected=True):
     if not master_ips:
         return
 
+    pub_port = None
```

Comment:
I'm trying to fix the exception that is caused by calling __salt__ in utils/minion.py:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/salt/utils/schedule.py", line 837, in handle_func
    ret['return'] = self.functions[func](*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/salt/modules/status.py", line 1058, in master
    connected_ips = _connected_masters()
  File "/usr/lib/python2.7/site-packages/salt/utils/minion.py", line 65, in connected_masters
    config_port = __salt__['config.get']('publish_port')
NameError: global name '__salt__' is not defined
```
I do not know how to make that call work.
What I can do is set the parameter of connected_masters to None

```
connected_masters(pub_port=None):
```
so that there is no conflict with it being called from somewhere else.